testdata/stackdriver/client_access_log_entry_sampled.yaml.tmpl

```diff
@@ -0,0 +1,27 @@
+http_request:
+  request_method: "GET"
+  request_url: "http://127.0.0.1:{{ .Ports.ClientPort }}/{{ .Vars.RequestPath }}"
+  server_ip: "127.0.0.1:{{ .Ports.ClientPort }}"
+  protocol: "http"
+  status: {{ .Vars.SDLogStatusCode }}
+labels:
+  destination_principal: "{{ .Vars.DestinationPrincipal }}"
+  destination_service_host: server.default.svc.cluster.local
+  response_flag: "-"
+  service_authentication_policy: {{ .Vars.ServiceAuthenticationPolicy }}
+  destination_name: productpage-v1-84975bc778-pxz2w
+  destination_namespace: default
+  source_principal: "{{ .Vars.SourcePrincipal }}"
+  destination_workload: productpage-v1
+  destination_app: productpage
+  destination_canonical_service: productpage-v1
+  destination_canonical_revision: version-1
+  destination_version: v1
+  protocol: http
+  log_sampled: "true"
+  upstream_cluster: "outbound|9080|http|server.default.svc.cluster.local"
```

Comment:
why does this entry not mention any trace related information?

Reply:
We are not sending any request with trace headers... will add a test for that in a separate pr